src/Pages/Home/components/TopLeftAlertMenu.tsx

```diff
@@ -5,35 +5,43 @@ import { RightArrowSvg } from "../../../components/Icons/RightArrow"
 import { ListOfAlerts } from "./ListOfAlerts"
 import { ArrowTopSvg } from "../../../components/Icons/ArrowTop"
 import { OpenArrowsSvg } from "../../../components/Icons/OpenArrowsSvg"
+import { MuteAlertSvg } from "../../../components/Icons/MuteAlert"
+import { useTranslation } from "react-i18next"
 
 export function TopLeftAlertMenu() {
+    const {t} = useTranslation();
+    
     const [isAlertOpen, setIsAlertOpen] = useState(false)
+    const [isAlertMuted, setIsAlertMuted] = useState(false)
+    const [unreadAlerts] = useState(0)
```

Comment:
this line is not needed cause we will have this from a query please delete if can

Reply:
Done
See the PR in the internal network.